packages/sdk/src/utils/cache.ts

```diff
@@ -1,13 +1,14 @@
 import { Token } from "../entities/token";
 import { CACHER } from "../commons";
 import { warn } from "./invariant";
+import { Address } from "viem";
```

Comment:
Here too, `type`.

Reply:
Addressed in 45e90a7916296a0f62790b211aee43c50e106129